package.xml

```diff
@@ -0,0 +1,24 @@
+<?xml version="1.0"?>
+<?xml-model href="http://download.ros.org/schema/package_format3.xsd" schematypens="http://www.w3.org/2001/XMLSchema"?>
+<package format="3">
+  <name>buoy_msgs</name>
+  <version>0.0.0</version>
+  <description>Power Buoy messages (telemetry) and services (commands)</description>
+  <maintainer email="anderson@mbari.org">Michael Anderson</maintainer>
+  <license>TODO: License declaration</license>
+
+  <buildtool_depend>ament_cmake</buildtool_depend>
+  
+  <build_depend>rosidl_default_generators</build_depend>
+  <exec_depend>rosidl_default_runtime</exec_depend>
+  <member_of_group>rosidl_interface_packages</member_of_group>
```

Comment:
The ament linters are picky with the order things in `package.xml`. Running `colcon test` fails with:

```
[0.857s] 3: File 'package.xml' is invalid:
[0.857s] 3: /media/chapulina/TERA/code/buoy_msgs/package.xml:16: element depend: Schemas validity error : Element 'depend': This element is not expected. Expected is one of ( member_of_group, export ).
[0.857s] 3: /media/chapulina/TERA/code/buoy_msgs/package.xml fails to validate
```

That's fixed by moving `member_of_group` below `test_depend`

Reply:
fixed